content/blog/2020-09-01-august-release/contents.lr

```diff
@@ -0,0 +1,119 @@
+title: InvenioRDM August Release
+---
+author: Sara Gonzales, Guillaume Viger
+---
+body:
+
+We are happy to announce InvenioRDM Alpha 10 (August release)! Thank you to our team members for their efforts on this release.
+
+### What's new?
+
+With less resources than usual, we made some limited changes this month. However, three key areas *were* tackled.
```

Comment:
It's true that there were fewer resources in the InvenioRDM team, but there was an important number of people joining Invenio Sprint, which brought new features to RDM as well :)

Reply:
But @zzacharo :muscle: was on vacation part of the time: he's a big deal ;)

Reply:
that's true, but if you look at it on the other way, there +10 people working on Invenio this month :)

Reply:
Haha ok, I've removed it :grin: